### DISCUSSION TITLE
Convert covers to WebP, how?

### DISCUSSION THREAD

**Discussion by pixelfabs** [adityatelange/hugo-PaperMod#539] [Q&A] [answered]
I like to convert my covers to WebP. WebP is supported since Hugo 0.83. I had a look at `layout/partials/covers.html` and assume it should be added to line 16 of `covers.html`

`{{ printf "%s %s" (($cover.Resize (printf "%sx" $size)).Permalink) (printf "%sw ," $size) -}}`

[Hugo Docs](https://gohugo.io/content-management/image-processing/#target-format) says:

`{{ $image.Resize "600x webp" }}`

But I have no idea "where" I should add webp to `covers.html`.

Can someone point me in the right direction?

**Comment by danielfdickinson** [comment, marked as answer]
Replace: ``{{ printf "%s %s" (($cover.Resize (printf "%sx" $size)).Permalink) (printf "%sw ," $size) -}}`` with ``{{ printf "%s %s" (($cover.Resize (printf "%sx webp" $size)).Permalink) (printf "%sw ," $size) -}}``

Which reminds me to make sure I did not forget to make that change in a personal fork after I did updates for sending upstream (here).

**Reply by pixelfabs** [reply]
Excellent! Thank man :-)
It works just as intended.
